docs/source/pca.rst

```diff
@@ -142,6 +142,46 @@ One can use the ``fit`` method to perform PCA over a given dataset.
     # reconstruct testing observations (approximately)
     Xr = reconstruct(M, Yte)
 
+**Example with iris dataset and plotting:**
+
+.. code-block:: julia
+
+    using MultivariateStats, RDatasets, Plots
+    plotly() # using plotly for 3D-interacive graphing
+
+    # load iris dataset 
+    iris = dataset("datasets", "iris")
+
+    # split half to training set
+    Xtr = convert(Array,DataArray(iris[1:2:end,1:4]))'
+    Xtr_labels = convert(Array,DataArray(iris[1:2:end,5]))
+
+    # split other half to testing set
+    Xte = convert(Array,DataArray(iris[2:2:end,1:4]))'
+    Xte_labels = convert(Array,DataArray(iris[2:2:end,5]))
+
+    # suppose Xtr and Xte are training and testing data matrix,
+    # with each observation in a column
+
+    # train a PCA model
+    M = fit(PCA, Xtr; maxoutdim=100)
```

Comment:
You can just call `M = fit(PCA, Xtr, maxoutdim=3)` to get first 3 principal components.

Reply:
OK, just updated